src/lib/y2network/connection_config/base.rb

```diff
@@ -204,6 +204,23 @@ def static?
         bootproto ? bootproto.static? : true
       end
 
+      # Return the first hostname associated with the primary IP address
+      #
+      # @return [String, nil]
```

Comment:
please document when nil can happen. And originally it cannot return nil. So what it was previously? empty string?

Reply:
Previously was initialized to nil

Reply:
https://github.com/yast/yast-network/pull/1128/files#diff-6b7270d3f0aa0539b5197b7ed2502dbcd53c7549dd2d701d3251f4b7c5faf49eL127